src/Entity/EntityDocument.php

```diff
@@ -31,4 +31,17 @@ public function getId();
 	 */
 	public function getType();
 
+	/**
+	 * Sets the id of the entity. If the id is not of the correct type,
+	 * an exception will be thrown. A specific derivative of EntityId is
+	 * always supported.
+	 *
+	 * @since 3.0
+	 *
+	 * @param EntityId $id
+	 *
+	 * @throws \InvalidArgumentException
```

Comment:
I merely did not re-apply it since I don't see reason to. Feel free to change it if you want though